packages/forms-web-app/src/dynamic-forms/has-questionnaire/route.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const { renderTaskList } = require('../controller');
+
+const router = express.Router();
+
+router.get('/questionnaire/:caseRef', renderTaskList);
```

Comment:
So it's a little annoying, we have a file: /lib/views.js
It's defines the path to the nunjucks files for views but is also used for urls
This hard ties the file path for a view to the routing (blocking easy sharing of views between different routes)
I tried to split that at some point and ended up with hundreds of changes so gave up.

The other tricky bit is that the routes are split up between nested route files
So the whole path to this is actually /manage-appeals/questionnaire/caseRef
The base url is in a const in that view file so: manage-appeals/questionnaire
Leaving as is obviously means that if we change the url in views.js it does not update this path, but at least it's only 1 place and links to it will remain the same

Reply:
Ah, ok. I did spot that file and nearly added another comment to ask why we're using a view location as a URL but I thought it would be something like that. So, in short, we do have the stem of the URL in a constants file but that can't be used in this instance as we're adding the route in a nested router.